pkg/addonfactory/addondeploymentconfig.go

```diff
@@ -22,6 +22,16 @@ var AddOnDeploymentConfigGVR = schema.GroupVersionResource{
 	Resource: "addondeploymentconfigs",
 }
 
+type resourceRequirements struct {
+	Limits   map[string]string `json:"limits,omitempty"`
+	Requests map[string]string `json:"requests,omitempty"`
+}
+
+type regexResourceRequirements struct {
+	ContainerIDRegex string               `json:"containerIDRegex"`
```

Comment:
can we add some comment on what ContainerIDRegex means

Reply:
Comments added.